benches/benchmark.rs

```diff
@@ -0,0 +1,28 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use overpunch::{convert_from_signed_format, convert_to_signed_format};
+use rust_decimal::Decimal;
+
+fn bench_convert_from_signed_format(c: &mut Criterion) {
+    c.bench_function("convert_from_signed_format", |b| {
+        b.iter(|| {
+            convert_from_signed_format("123{", "s9(7)v99");
```

Comment:
Suggest: using [black box](https://docs.rs/criterion/latest/criterion/fn.black_box.html) here to prevent the compiler from optimizing and reducing the effectiveness of the benching.

Something like:
```rust
let input_data = criterion::black_box("123{");
let format_spec = criterion::black_box("s9(7)v99");
    
convert_from_signed_format(input_data, format_spec);
```